ocean_provider/file_types/definitions.py

```diff
@@ -72,7 +72,10 @@ def check_details(self, with_checksum=False):
                 if result:
                     status_code = result.status_code
                     headers = copy.deepcopy(result.headers)
-                    files_url = copy.deepcopy(result.url)
+                    if "arweave" in result.url:
```

Comment:
what if arweave gateway url is  "http://gateway.ar.example" ?
honestly, I would move this to Arweave class  (overwrite check_details) and do it for every arweave file

Reply:
Sure, thanks for pointing out.